script/tool/test/native_test_command_test.dart

```diff
@@ -122,11 +122,9 @@ void main() {
       final Directory pluginExampleDirectory =
           pluginDirectory1.childDirectory('example');
 
-      // Exit code 66 from testing indicates no tests.
-      final MockProcess noTestsProcessResult = MockProcess();
-      noTestsProcessResult.exitCodeCompleter.complete(66);
       processRunner.mockProcessesForExecutable['xcrun'] = <io.Process>[
-        noTestsProcessResult,
+        // Exit code 66 from testing indicates no tests.
+        MockProcess(exitCode: 66),
```

Comment:
nit: It would be nice if we could link to some documentation that says 66 means this.  I'm not sure looking at the text output would be less fragile.

Reply:
I don't have a reference on that; I didn't write the `xctest` logic. This is just adjusting the way that exit code is being returned, not the use of the exit code or the comment.